### ISSUE TITLE
PDF reports are not being properly generated on Windows (Trac #984)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#1055]
4.1.1 is generating empty PDF reports on W8.1 (tested by gonzalezm) & W10 (tested by smk78 & butler), but reports are fine on MacOS 10.11 (tested by wojciech).

Win log file reports:

2017-08-02 10:30:19,621 ERROR Report string expected 'name: value' but got u'
2017-08-02 10:30:52,127 ERROR Error creating pdf: No module named common

Problem was not present in 4.1. Though, interestingly, the log file still gives the first (but not the second) error above...

Migrated from http://trac.sasview.org/ticket/984
```json
{
    "status": "closed",
    "changetime": "2017-09-15T16:10:32",
    "_ts": "2017-09-15 16:10:32.431373+00:00",
    "description": "4.1.1 is generating empty PDF reports on W8.1 (tested by gonzalezm) & W10 (tested by smk78 & butler), but reports are fine on MacOS 10.11 (tested by wojciech).\n\nWin log file reports:\n\n2017-08-02 10:30:19,621 ERROR Report string expected 'name: value' but got u'\n2017-08-02 10:30:52,127 ERROR Error creating pdf: No module named common\n\nProblem was not present in 4.1. Though, interestingly, the log file still gives the first (but not the second) error above...\n",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2017-08-02T09:54:48",
    "component": "SasView",
    "summary": "PDF reports are not being properly generated on Windows",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by smk78** [comment]
Trac update at `2017/08/02 11:25:41`: **smk78** commented:

krzywon notes:

Looking through the logs of the Windows build for the 4.1.1 release, the xhtml2pdf package for generating pdf files was never bundled. This is probably why the PDFs aren't being created. The OSX job bundled the package as expected.

**Comment by butlerpd** [comment]
Trac update at `2017/08/27 04:10:01`: 
* **butler** commented:

There was a bit more to it including a path typo.  Fixed in branch 4.1.1_path_issues and pulled into 4_1_issues and eventually released as 4.1.2

4.1.2 has been tested and PDF now works again as does master builds though 4.1.1 and 4.1.2 changes have not yet been merged into master.  Nonetheless am closing this ticket as fixed


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "new" to "closed"
